src/LocalField/neq.jl

```diff
@@ -0,0 +1,216 @@
+#export: degree_relative, random_elem, one_root, norm_equation  
+
+
+degree_relative(L::Hecke.LocalField,K::Hecke.LocalField) = Int(absolute_degree(L)//absolute_degree(K)
+)
+
+ degree_relative(L::Hecke.RelFinField,K:: Hecke.RelFinField)  = Int(absolute_degree(L)//absolute_degree(K))
+degree_relative(L::Hecke.RelFinField,K::FqNmodFiniteField)  = Int(absolute_degree(L)//absolute_degree(K))
+
+degree_relative(L::FqNmodFiniteField, K::Hecke.Nemo.GaloisField)  = Int(absolute_degree(L)//absolute_degree(K))
+
+
+##############################################
+#random element with small coefficients
+##############################################
+
+function random_elem(L::Hecke.LocalField)
```

Comment:
Wouldn't it make more sense to turn this into a `rand` method?

Reply:
The field is with infinite elements. So I wanted to construct a random element with small coefficients. One can change 5 with any other number